### PR TITLE
Fix LGTM alerts

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

File: vpn_slice/__main__.py [hunk 1]
```diff
@@ -434,8 +434,8 @@ def parse_args_and_env(args=None, environ=os.environ):
     p = argparse.ArgumentParser()
     p.add_argument('routes', nargs='*', type=net_or_host_param, help='List of VPN-internal hostnames, included subnets (e.g. 192.168.0.0/24), excluded subnets (e.g. %%8.0.0.0/8), or aliases (e.g. host1=192.168.1.2) to add to routing and /etc/hosts.')
     g = p.add_argument_group('Subprocess options')
-    p.add_argument('-k','--kill', default=[], action='append', help='File containing PID to kill before disconnect (may be specified multiple times)')
-    p.add_argument('-K','--prevent-idle-timeout', action='store_true', help='Prevent idle timeout by doing random DNS lookups (interval set by $IDLE_TIMEOUT, defaulting to 10 minutes)')
+    g.add_argument('-k','--kill', default=[], action='append', help='File containing PID to kill before disconnect (may be specified multiple times)')
+    g.add_argument('-K','--prevent-idle-timeout', action='store_true', help='Prevent idle timeout by doing random DNS lookups (interval set by $IDLE_TIMEOUT, defaulting to 10 minutes)')
```
Comment (dlenski) [inline vpn_slice/__main__.py:434]: Good catch, thanks :+1:

**Review by dlenski — CHANGES_REQUESTED** [review]
This all LGTM, _except_ for the removal of the `else` clauses from loops. I happen to prefer these stylistically to keep the post-loop clean-up/finish-up code visually associated with the loop in many cases :grin:

File: vpn_slice/__main__.py (hunk 1 above)
Comment (DimitriPapadopoulos) [reply vpn_slice/__main__.py:434]: OK, then I shall revert the for/else change.

Comment (dlenski) [reply vpn_slice/__main__.py:437]: Thank you, merging!!

**Review by dlenski — APPROVED** [review]

**Pull request merged**